DoctorWho.Db/Repositories/CompanionRepository.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Dapper;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System.Data;
+
+namespace DoctorWho.Db.Repositories
+{
+    public class CompanionRepository
+    {
+        private readonly DoctorWhoCoreDbContext _context;
+        private readonly string _connectionString;
+
+        public CompanionRepository(DoctorWhoCoreDbContext context)
+        {
+            _context = context;
+             _connectionString = "Server=(localdb)\\MSSQLLocalDB;Database=DoctorWhoCore;Trusted_Connection=True;";
```

Comment:
I don't think Connection string is necessary here, did you try without it? since you already passed it to Context